js/apps/system/_admin/aardvark/APP/frontend/js/lib/tile.stamen.js

```diff
@@ -44,7 +44,7 @@ var SUBDOMAINS = "a. b. c. d.".split(" "),
         }
     };
 
-PROVIDERS["terrain-classic"].url = "http://{S}tile.stamen.com/terrain/{Z}/{X}/{Y}.png";
+PROVIDERS["terrain-classic"].url = "//stamen-tiles-{S}a.ssl.fastly.net/terrain/{Z}/{X}/{Y}.png";
```

Comment:
Is there any reason not to always use https rather than protocol-relative? Does the server even serve plaintext HTTP?

Reply:
That's just how it's officially being used.

Reply:
Fine by me, but I think this should probably be https since otherwise it will use http unless aardvark is accessed over https, which could pose minor security implications.